diff --git a/node_modules/next/dist/build/swc/options.js b/node_modules/next/dist/build/swc/options.js
index 4d9e6fc..77e9c6d 100644
--- a/node_modules/next/dist/build/swc/options.js
+++ b/node_modules/next/dist/build/swc/options.js
@@ -48,7 +48,7 @@ function getParserOptions({ filename, jsConfig, ...rest }) {
         ...rest,
         syntax: hasTsSyntax ? "typescript" : "ecmascript",
         dynamicImport: true,
-        decorators: enableDecorators,
+        decorators: true,
         // Exclude regular TypeScript files from React transformation to prevent e.g. generic parameters and angle-bracket type assertion from being interpreted as JSX tags.
         [hasTsSyntax ? "tsx" : "jsx"]: !isTSFile,
         importAssertions: true
@@ -91,6 +91,7 @@ function getBaseSWCOptions({ filename, jest, development, hasReactRefresh, globa
                     }
                 } : {},
                 legacyDecorator: enableDecorators,
+                decoratorVersion: "2022-03",
                 decoratorMetadata: emitDecoratorMetadata,
                 useDefineForClassFields: useDefineForClassFields,
                 react: {
